pyproject.toml

```diff
@@ -3,9 +3,7 @@ name = "opensquirrel"
 version = "0.0.5"
 description = "A quantum circuit transformation and manipulation tool"
 authors = [
-    "Pablo Le Henaff <p.lehenaff@tudelft.nl>",
-    "Roberto Turrado Camblor <r.turradocamblor@tudelft.nl>",
-    "Chris Elenbaas <chris.elenbaas@tno.nl>"
+    "Quantum Inspire <support@quantum-inspire.com>"
```

Comment:
+Juan +Stan?

Reply:
Oh sorry, you removed the individual names, OK!

Reply:
No specific authors will be defined in the project files.